### PR TITLE
Add more v9 information to README and to northstar docs

### PR DESCRIPTION
**Pull request by jurokapsiar**
- Move information about v9 to the first column of table in Readme - [preview](https://github.com/jurokapsiar/fluentui/tree/v9-readme)
- Move react-northstar information to its separate readme - [preview](https://github.com/jurokapsiar/fluentui/tree/v9-readme/packages/fluentui)
- Add explanation for the status of react-northstar
- Promote v9 on react-northstar docs homepage

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/2cf6f6338447c0e81dd06763e4791972a34b0548'>2cf6f6338447c0e81dd06763e4791972a34b0548</a></sub>
<div id="bundleSizeComment9423"/>

**Review by chpalac — APPROVED** [review]
LGTM

File: packages/fluentui/README.md [hunk 1]
```diff
@@ -29,25 +33,28 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for a step-by-step setup and development
...
 ### What's happening with Teams support?
 
-`@fluentui/react-northstar v0.x.x` (formerly `@fluentui/react v0.x.x`) has been the official set of components for Teams. This is still the case and no support is being dropped.
+`@fluentui/react-northstar v0.x.x` (formerly `@fluentui/react v0.x.x`) has been the official set of components for Teams. This is still the case and no support is being dropped. New features will only be added to Fluent UI React Components v9.
 
 [1]: https://github.com/microsoft/fluent-ui-react
 [2]: https://github.com/microsoft/fluentui/blob/master/packages/fluentui/CHANGELOG.md
 [3]: https://github.com/microsoft/fluentui/issues/new/choose
+[4]: https://github.com/microsoft/fluentui/issues?q=is%3Aissue+is%3Aopen+label%3A%22Fluent+UI+react-northstar+%28v0%29%22
+[5]: https://react.fluentui.dev/?path=/docs/concepts-migration-overview--page
```
Comment (JustSlone) [inline packages/fluentui/README.md:60]: This link will probably break in the near future when Geoff redos the table of contents, I'll leave it you whether you want to just link to the doc site and say "See Migrations Docs on v9 documentation site"

**Review by JustSlone — APPROVED** [review]
Love it, thanks!

**Comment by paulgildea** [comment]
This looks good to me! Thanks @jurokapsiar !

File: packages/fluentui/README.md [hunk 2]
```diff
@@ -29,25 +33,28 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for a step-by-step setup and development
...
 
 See the doc site at https://aka.ms/fluent-ui.
 
 ## FAQs
 
-### Why the move?
+### What is the future of Fluent UI React Northstar?
 
-We're in the process of converging UI projects at Microsoft under the "Fluent UI" banner to simplify the dev story. In the process, we're making performance improvements and enabling greater interoperability of these components across more places and platforms at Microsoft.
+The future of `@fluentui/react-northstar` is called `@fluentui/react-components`. It builds on top of many base principles that were present in Northstar, streamlines the API and brings significanlty improved performance.
```
Comment (khmakoto) [inline packages/fluentui/README.md:46]:
```suggestion
The future of `@fluentui/react-northstar` is called `@fluentui/react-components`. It builds on top of many base principles that were present in Northstar, streamlines the API and brings significantly improved performance.
```

**Review by khmakoto — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -6,33 +6,33 @@
 
 Fluent UI web represents a collection of utilities, React components, and web components for building web applications.
```
Comment (layershifter) [inline README.md:7]: image

Is there sense to remove this?

**Review by layershifter — APPROVED** [review]

File: packages/fluentui/README.md (hunk 1 above)
Comment (jurokapsiar) [reply packages/fluentui/README.md:60]: we'll fix it then - I think having the links is useful.

File: README.md (hunk 3 above)
Comment (jurokapsiar) [reply README.md:7]: I will follow up on this with @paulgildea - for now let's keep it in.

**Pull request merged**